src/styles/blocks/main-contant.scss

```
@@ -2,14 +2,13 @@
   width: 100%;
 
   &__container {
-    display: flex;
-    justify-content: space-between;
-    flex-wrap: wrap;
+    display: grid;
     max-width: 944px;
+    grid-template-columns: repeat(auto-fill, 200px);
```

```suggestion
    grid-template-columns: repeat(4, 200px);
```
можно задать количество колонок как в задании и тогда не надо  max-width: .